src/ImageSharp/Primitives/ColorMatrix.cs

```diff
@@ -204,7 +204,7 @@ public bool IsIdentity
         /// <returns>The resulting matrix.</returns>
         public static ColorMatrix operator +(ColorMatrix value1, ColorMatrix value2)
         {
-            ColorMatrix m;
+            var m = default(ColorMatrix);
```

Comment:
I don't like this. The declaration was more clear and less verbose before. Why the change?

Reply:
because of https://docs.microsoft.com/en-us/dotnet/csharp/misc/cs0818

Reply:
That's not the same thing. Our struct was explicitly declared. I'll leave it for now anyway.